barretenberg/cpp/src/barretenberg/honk/proof_system/protogalaxy_prover.hpp

```diff
@@ -32,7 +32,7 @@ template <class ProverInstances_> class ProtoGalaxyProver_ {
     using RelationEvaluations = typename Flavor::TupleOfArraysOfValues;
 
     ProverInstances instances;
-    ProverTranscript<FF> transcript;
+    BaseTranscript<FF> transcript;
```

Comment:
currently no structure for Protogalaxy prover, but Mara mentioned that might be useful?